usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
I agree with Lubos, there should be place for "common" tests (log-in, tasks, users,...)